### ISSUE TITLE
Race condition crash when using Model with varying properties

### ISSUE THREAD

**Issue by JVillella** [brianzinn/react-babylonjs#79]
There seems to be a race condition between setting the `LoadedModel.rootMesh` in `onMount` and applying positions on the rootMesh (target) in `applyUpdateToInstance`.

IOW, we're finding 

`ModelLifecycleListener.onMount` is not always being called before,

```js
var applyUpdateToInstance = function (hostInstance, update, type) {
    var target = update.target !== undefined ? hostInstance[update.target] : hostInstance;
    switch (update.changeType) {
        // ...
        case PropChangeType.Vector3:
            if (target && target[update.propertyName]) {  // <== CRASH
                target[update.propertyName].copyFrom(update.value);
            }
```

We're not doing anything fancy with the usage. We have a `Model`,

```js
    return <Model
      key={id}
      name={id}
      rootUrl={rootUrl}
      sceneFilename={sceneFilename}
      rotation={rotation}
      position={position}>
      <PBRMaterial
        name="material"
        microSurface={0.96}
        reflectivityColor={new Color3(1, 0, 0)}
        albedoColor={new Color3(1, 0, 0)} />
    </Model>
```

where the `rotation` and `position` change frame to frame.

**Comment by brianzinn** [comment]
hi @JVillella - thanks for reporting.   I tried to follow through your sample, but am unable to reproduce - can you share a reproduction of the issue?  I am currently on vacation, but with a repro can hopefully get a fix out right away.  Cheers.

**Comment by brianzinn** [comment]
This is fixed in 3.0.  Note there are some breaking changes - notably with Model that you need to wrap in `<Suspense />`.
